examples/signalfx/signalfx-collector.yaml

```diff
@@ -0,0 +1,78 @@
+extensions:
+  health_check:
+  zpages:
+receivers:
+  otlp:
+  sapm:
+  signalfx:
+  # This section is used to collect OpenTelemetry metrics
+  # Even if just a SignalFx µAPM customer, these metrics are included
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: 'otel-collector'
+          scrape_interval: 10s
+          static_configs:
+            - targets: ['localhost:8888']
+              # If you want to use the environment filter
+              # In the SignalFx dashboard
+              #labels:
+                #environment: demo
+          metric_relabel_configs:
+            - source_labels: [ __name__ ]
+              regex: '.*grpc_io.*'
+              action: drop
+  # Enable Zipkin to support Istio Mixer Adapter
+  # https://github.com/signalfx/signalfx-istio-adapter
+  zipkin:
+processors:
+  batch:
```

Comment:
Should we add blank line after each component? Would help visually associate comments with components.

Reply:
hum is the lack of black lines so this can be pasted directly into the k8s config?